### PR TITLE
vaccinations tile

### PR DESCRIPTION
**Pull request by samMcCarthyBJSS**
Story [AB#10814](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/10814)
Task [AB#10859](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/10859)

image

### CODE REVIEW

**Review by bjss-chris — COMMENTED** [review]
Just one question.

File: app/pages/homepage/manifest.json [hunk 1]
```diff
@@ -7,14 +7,16 @@
...
-  "viewSolutionsPromoHeading": "GP IT Futures framework",
-  "viewSolutionsPromoDescription": "Find and compare Catalogue solutions available on the GP IT Futures framework",
   "adminPromoHeading": "Manage organisations and users",
-  "adminPromoDescription": "Add new organisations and user accounts or edit existing details.",
+  "adminPromoDescription": "Add new organisations and user accounts or edit existing details",
   "covid19PromoHeading": "Reduce GP visits",
-  "covid19PromoDescription": "Find Catalogue Solutions that can help prevent the spread of coronavirus.",
+  "covid19PromoDescription": "Find Catalogue Solutions that can help prevent the spread of coronavirus",
+  "guidePromoHeading": "Buyer’s Guide",
+  "guidePromoDescription": "Terminology you’ll need to know when procuring Catalogue Solutions.",
```
[the remark this answers is not in the document]
Comment (samMcCarthyBJSS) [reply app/pages/homepage/manifest.json:15]: yeah there's been an update to the style guide
luke was making a new ticket for it but thought I'd remove them from here while I was on the page

[the remark this answers is not in the document]
Comment (bjss-chris) [reply app/pages/homepage/manifest.json:15]: Do you need to remove this one (on line 15) as well then?

**Review by bjss-chris — APPROVED** [review]
👍

**Pull request merged**